### ISSUE TITLE
Changing @tab-content__border variable has no effect in Blank theme

### ISSUE THREAD

**Issue by denistrator** [magento/magento2#14999]
### Preconditions
Any Magento 2
FE theme inherited from Magento/Blank

### Steps to reproduce
In `Custom/theme/web/css/source/_theme.less` set less variables to:
@tab-content__border-top-status: false;
@tab-content__border: 1px solid #000;

### Expected result
Tabs content has border

### Actual result
Tabs content has no border

### What causes the issue
[web/css/source/_sections.less](https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/blank/web/css/source/_sections.less#L24) calls `.lib-data-tabs()` mixin with `@_tab-content-border-top-status: true` argument, so `@tab-content__border-top-status` variable isn't used in  `.lib-data-tabs()`

### Solution
1. Remove `@_tab-content-border-top-status: true` argument in [web/css/source/_sections.less](https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/blank/web/css/source/_sections.less#L24) (as it's done in [Magento/Luma](https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/luma/web/css/source/_sections.less#L66));
2. Add `@tab-content__border-top-status: true` in [web/css/source/_variables.less](https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/blank/web/css/source/_variables.less)

### Workaround
[Create](https://devdocs.magento.com/guides/v2.0/frontend-dev-guide/css-guide/css_quick_guide_approach.html#structured_changes) `Custom/theme/web/css/source/_sections_extend.less` with:

```
.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
    .product.data.items {
        > .item.content {
            ._lib-tab-content-border(
                @_tab-content-border-top-status: @tab-content__border-top-status
            );
        }
    }
}
```

**Comment by hitesh-wagento** [comment]
I am working on this

**Comment by sidolov** [comment]
Hi @denistrator. Thank you for your report. 
The issue has been fixed in magento/magento2#15917 by @hitesh-wagento in 2.1-develop branch
Related commit(s):
 - [5b34517ffada38f95a2ec0deae9e20a6b628ab4a](https://github.com/magento/magento2/commit/5b34517ffada38f95a2ec0deae9e20a6b628ab4a)

The fix will be available with the upcoming 2.1.15 release.

**Comment by magento-engcom-team** [comment]
Hi @denistrator. Thank you for your report. 
The issue has been fixed in magento/magento2#15916 by @hitesh-wagento in 2.3-develop branch
Related commit(s):
 - [21ce9fb9ae8696d24343058a981aaa18b8bc16d8](https://github.com/magento/magento2/commit/21ce9fb9ae8696d24343058a981aaa18b8bc16d8)

The fix will be available with the upcoming 2.3.0 release.

**Comment by magento-engcom-team** [comment]
Hi @denistrator. Thank you for your report. 
The issue has been fixed in magento/magento2#15914 by @hitesh-wagento in 2.2-develop branch
Related commit(s):
 - [4877e4cd75b5bdb594bce95d9aaae39865dbb43e](https://github.com/magento/magento2/commit/4877e4cd75b5bdb594bce95d9aaae39865dbb43e)

The fix will be available with the upcoming 2.2.6 release.
